README.md

```diff
@@ -7,30 +7,45 @@
 What is webpki?
 ==================
 
-webpki is a library that validates Web PKI (TLS/SSL) certificates. webpki
-is designed to provide a **full** implementation of the client side of the
-**Web PKI** to a diverse range of applications and devices,
-including embedded (IoT) applications, mobile apps, desktop applications, and
-server infrastructure. webpki is intended to not only be the best
-implementation of the Web PKI, but to also *precisely define* what the Web PKI
-is.
+webpki is a library that validates Web PKI (TLS/SSL) certificates. It's
+used by [Rustls](https://github.com/rustls/rustls) to handle certificate related
+tasks required for implementing TLS clients and servers. It's scope includes:
+
+* Representing trust anchors - webpki allows instantiating "root" certificate
+  authorities that are implicitly trusted because they are distributed by
+  software you galready trust (like a web browser, an operating system, or
+  curated by a trusted project like [webpki-roots](https://github.com/rustls/webpki-roots)).
+
+* Parsing certificates - webpki can convert from the raw encoded form of
+  a certificate into something that can be used for making trust decisions.
+
+* Path building - webpki can determine if a certificate for an end-entity like
```

Comment:
Is this the expected hyphenation for end-entity?

Reply:
Fixed.